<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OneModelWidget</class>
 <widget class="QWidget" name="OneModelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="m_pBtnShowOrHide">
        <property name="text">
         <string>展示</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_pLabName">
        <property name="text">
         <string>名字</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_pBtnExpand">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_pWgtFunction" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="m_pBtnShowBoundingBox">
        <property name="text">
         <string>展示包围盒</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="m_pBtnShowGrid">
        <property name="text">
         <string>展示线框模型</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="m_pBtnLoopSubdivision">
        <property name="text">
         <string>进行一次Loop模型细分</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
